src/routes/+page.md

```diff
@@ -114,6 +114,28 @@ To use it, drop a `<PaymentElement>` component in your form:
 </form>
 ```
 
+When you want to set default values as per the [Stripe Payment Element Options `defaultValues` object](https://stripe.com/docs/js/elements_object/create_payment_element#payment_element_create-options) you do so by the following:
```

Comment:
I wonder if this is needed?

Maybe it would be better to just add the `options` attribute to the example that preceeds this?
It feels like a lot text to describe on option.


Maybe something like:

```
<PaymentElement options={...}/>
``` 

Eventually, I'd like to have all attributes listed somewhere in the docs. But for now, I'd prefer to optimize for short docs.

Otherwise, everything looks good!

Reply:
Sure man, I see what you mean. it does waffle a little. Ill get it removed